src/main/java/com/microsoft/azure/relay/ClientWebSocket.java

```diff
@@ -247,6 +247,7 @@ else if (data instanceof String) {
 							}
 							
 							int bytesToSend = bytes.remaining();
+							// sendBinary() will cause the content of the byte array within the ByteBuffer to change
```

Comment:
>content of the byte array [](start = 38, length = 25)

This is weird. What do the array contents become?

Reply:
Becomes garbage values...I haven't looked into it, not sure what they are doing with the array after sending

---
In reply to: [294013165](https://github.com/Azure/azure-relay-java/pull/47#discussion_r294013165) [](ancestors = 294013165)

Reply:
no reason, I'm just surprised they party on someone else's buffer

---
In reply to: [294013165](https://github.com/Azure/azure-relay-java/pull/47#discussion_r294013165) [](ancestors = 294013165)